cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -390,6 +392,181 @@ func TestAddFlags(t *testing.T) {
 	}
 }
 
+func TestApplyTo(t *testing.T) {
+	fs := pflag.NewFlagSet("addflagstest", pflag.ContinueOnError)
+	s, _ := NewKubeControllerManagerOptions()
+	for _, f := range s.Flags([]string{""}, []string{""}).FlagSets {
```

Comment:
comments on what these args are would be helpful

Reply:
thanks for the review @liggitt! I've added a generic comment about these args being required for kube controller manager. Is that the one you are referring to?

Reply:
oh, I meant a comment explaining what the `[]string{""}, []string{""}` args were, but I see those were copied from the existing test